src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/DecisionMakerConsts.java

```diff
@@ -15,6 +15,10 @@
 
 package com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker;
 
+import com.google.gson.JsonParser;
+
 public class DecisionMakerConsts {
     public static final String CACHE_MAX_WEIGHT = "maximumWeight";
+
+    public static final JsonParser JSON_PARSER = new JsonParser();
```

Comment:
I can see why you want to put this single static parser in one place. Is this the right class for it though? Should we move it to some util?

Reply:
Right. Moving it to util is a good idea. Will merge this (unblock other build issues) and address this as a part of my next PR.

This issue will be resolved once we upgrade the dependency version.